src/transformStateWithClones.js

```diff
@@ -8,6 +8,37 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+
+  const array = [];
```

Comment:
Not the best name for an array. 
The name of the variable to which an array is assigned should give a clear description of what the array holds.